wire-schema-tests/src/commonTest/kotlin/com/squareup/wire/recipes/MarkdownHandlerTest.kt

```diff
@@ -68,6 +70,10 @@ class MarkdownHandlerTest {
                   |}
         """.trimMargin(),
       )
+      // We manually add fake runtime protos to please Wire when running on a non-JVM platforms.
+      // This isn't required if the code is to run on the JVM only.
+      add("google/protobuf/descriptor.proto".toPath(), "")
+      add("wire/extensions.proto".toPath(), "")
```

Comment:
This doesn't scale at all and prevent many tests from migration to `commonTest`.
In any case, we're not losing a feature by doing this but I wanna find a better way to support building test schemas in memory

Reply:
Agreed.